.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy to docker hub
+
+on:
+  pull_request:
+    branches:
+      - 'main'
```

Comment:
Won't this build the latest image once we merge into `main` ? is this what we want ? @owlas

Reply:
This is just on pull requests against main (and below I set `push` to false when those builds happen)

Reply:
Push to main would be:
```
on:
  push:
    branches:
      - 'main'
```